src/Components/Patient/ManagePatients.tsx

```diff
@@ -406,7 +406,9 @@ export const PatientManager = () => {
 
   const { data: permittedFacilities } = useQuery(
     routes.getPermittedFacilities,
-    {}
+    {
+      query: { limit: 1 },
```

Comment:
We're setting a limit of 1 result here, is this intentional?

Reply:
Yup, we are just using this to check if the user has access only to a single facility or more or not

Reply:
Makes sense